jena-arq/src/main/java/org/apache/jena/riot/system/RiotLib.java

```diff
@@ -107,6 +117,15 @@ public static boolean isBNodeIRI(String iri) {
         return iri.startsWith(bNodeLabelStart);
     }
 
+    /**
+     * Test whether a node is an ARQ-encoded blank node IRI.
+     */
+    public static boolean isBNodeIRI(Node node) {
+        if ( ! node.isURI() )
+            return false;
+        return isBNodeIRI(node.getURI());
```

Comment:
Or `return node.isURI() && isBNodeIRI(node.getURI())`

Reply:
Style choice. The `if` is a guard on the argument, then a test on the `getURI`. Think of it as lisp `(cond)` inspired‼️

I'm not a fan of (IMO) unnecessary brevity in compound expressions because multiple lines given multiple debug points.